src/interactions/abstractInteraction.ts

```diff
@@ -17,6 +17,34 @@ export module Interaction {
       this._componentToListenTo = component;
       this._hitBox = hitBox;
     }
+
+    /**
+     * Translates an <svg>-coordinate-space point to Component-space coordinates.
+     *
+     * @param {Point} p A Point in <svg>-space coordinates.
+     *
+     * @return {Point} The same location in Component-space coordinates.
+     */
+    protected _translateToComponentSpace(p: Point): Point {
+      var origin = this._componentToListenTo.originToSVG();
```

Comment:
"`svgOrigin`" sounds like it gets the origin *of the `<svg>`*, instead of the origin *relative to* the `<svg>`.